backend/service/github/github.go

```diff
@@ -40,12 +41,28 @@ const CurrentUser = ""
 
 type FileMap map[string]io.ReadCloser
 
+type StatsRoundTripper struct {
+	Wrapped http.RoundTripper
+	scope   tally.Scope
+}
+
+func (st *StatsRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
+	resp, err := st.Wrapped.RoundTrip(req)
+
+	if hdr := resp.Header.Get("X-RateLimit-Remaining"); hdr != "" {
+		if v, err := strconv.Atoi(hdr); err == nil {
+			st.scope.Gauge("github_rate_limit_remaining").Update(float64(v))
```

Comment:
whats the full stats namespace? I think its going to be `clutch.service.github.github_rate_limit_remaining` ? If so lets drop the extra github, you should be able to test this locally.